config/prod/kelsey-analysis.yaml

```diff
@@ -0,0 +1,21 @@
+template_path: managed-ec2-v5.yaml
```

Comment:
there is a newer version.  https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/3/files

please use the new version of this template.